README.md

```diff
@@ -27,3 +27,7 @@ about the current recursive call.
 Describe your reasoning and the conclusion you've come to. Your reasoning is the
 most important part. You do not need to prove that the invariant is correct. Add
 your answer to this markdown file.
+
+An invariant would the two numbers before the call fib(n). The function need access to the right values for the Fibonacci sequence for the two previous numbers in the list, fib(n-1) and fib(n-2). Each recursive call computes fib(n) by adding these two values, meaning that the recursive calls are moving toward the base case while maintaining this invariant until reaching fib(0) and fib(1).
```

Comment:
I don't know what you mean. "An invariant would the two numbers before the call" doesn't mean anything. What about the two numbers?